jquery.serializejson.js

```diff
@@ -15,6 +15,7 @@
     var serializedObject, formAsArray, keys, type, value, _ref, f, opts;
     f = $.serializeJSON;
     opts = f.optsWithDefaults(options); // calculate values for options {parseNumbers, parseBoolens, parseNulls}
+    $.extend(f.types, opts.types); // extend default types with custom types
```

Comment:
Here, instead of overriding the global types, it should just use the global types as defaults.

```javascript
opts.types = $.extend({}, opts.types, f.types); // set default types
```

This way, one could either modify the global types `$.serializeJSON.types['mynewglobaltype'] = function(str){...}` or define/override types just for the `$(selector).serializeJSON({types: {mynewlocaltype: function(str){...}}})` call, without affecting the global namespace (and have unexpected side effects).